lib/Parser/Parse.cpp

```diff
@@ -837,6 +837,14 @@ Symbol* Parser::AddDeclForPid(ParseNodePtr pnode, IdentPtr pid, SymbolType symbo
             }
             break;
         case knopVarDecl:
+
```

Comment:
Can you move this down to the switch statement just below that handles redeclarations? And do we not have issues involving function redeclaration errors in other situations? I wonder if it's necessary to have a fix that's specific to modules.